lib/aws.js

```diff
@@ -35,3 +35,53 @@ module.exports.getS3RVERConfiguration = function() {
         endpoint: new AWS.Endpoint('http://localhost:5000'),
     });
 };
+
+module.exports.uploadToS3 = async function (filePath, isDirectory, bucket, S3FilePath, localPath, callback) {
+    const s3 = new AWS.S3();
+
+    let body;
+    if (isDirectory) {
+        body = '';
+        S3FilePath += '/';
```

Comment:
What if the file already ends in `/`?

Reply:
Changed to `s3FilePath += s3FilePath.endsWith('/') ? '' : '/';`